clowdapp.yaml

```diff
@@ -83,6 +83,8 @@ objects:
         partitions: 1
       - topicName: platform.engine.results
         partitions: 1
+    testing:
+      iqePlugin: ros
```

Comment:
@digitronik Can you remove the following lines: https://github.com/RedHatInsights/ros-backend/pull/180/files#diff-0f518b5985b4ea86412d39c9757171411345db257cb061b84bb314a7a2b3d5e9R15-R17
and instead add `- engine` below `- rbac` here: https://github.com/RedHatInsights/ros-backend/pull/180/files#diff-0f518b5985b4ea86412d39c9757171411345db257cb061b84bb314a7a2b3d5e9R14

Reply:
cc @r14chandra

Reply:
gitlab connection failing.

Reply:
@apuntamb Is it because it's a private repo (for insights-engine)?

Reply:
@r14chandra I think yes. As we are using ext Jenkin this repo maybe not be accessible.

Reply:
Yes @digitronik. So, we need to use ci-int. Is there any thing to be considered for ci-int/ci-ext ?  @apuntamb

Reply:
@r14chandra @digitronik ! IMHO, we should first check if there is a way we can communicate with the internal repo using ci-ext, since it is a dependency.

Reply:
@r14chandra and @digitronik As you mentioned, we need to move to ci-int.
To consider the same, we need to update the webhook in github repo as well as update the job to ci-int in app-interface. Let me fix these two pre-requisites and check if the build is succeeding for th app-interface MR, and after that is merged, we can retest this PR.